pkgs/applications/editors/emacs/generic.nix

```diff
@@ -158,7 +158,7 @@ in stdenv.mkDerivation {
     '')
 
     (lib.optionalString nativeComp ''
-      wrapProgram $out/bin/emacs-* --prefix PATH : "${lib.makeBinPath [ binutils binutils-unwrapped ]}"
+      wrapProgram $out/bin/emacs-* --prefix PATH : "${lib.makeBinPath [ clang.bintools binutils binutils-unwrapped ]}"
```

Comment:
```suggestion
      wrapProgram $out/bin/emacs-* --prefix PATH : "${lib.makeBinPath [ stdenv.cc.bintools stdenv.cc.bintools.bintools ]}"
```

Reply:
This avoids pulling in clang when we aren't using it

Reply:
Awesome, that does look far better - I'll try this when my current build finishes.

Reply:
Done - I removed the `clang` and `binutils*` args above too, as they were made unnecessary by this change.